### PR TITLE
DOC-5710-ApplyReviewerChanges-IB2

### PR DESCRIPTION
**Pull request by ibsoln**
Further preening

### CODE REVIEW

File: modules/ROOT/examples/java/src/com/couchbase/code_snippets/Examples.java [hunk 1]
```diff
@@ -23,6 +23,12 @@
 /*      Credentials declared this way purely for expediency in this demo - use OAUTH in production code */
 private static final String DB_USER = "sync_gateway";
 private static final String DB_PASS = "password"; // <3>
+<<<<<<< Updated upstream
```
Comment (RichardSmedley) [inline modules/ROOT/examples/java/src/com/couchbase/code_snippets/Examples.java]: You need to edit down these two git diffs

Comment (ibsoln) [reply modules/ROOT/examples/java/src/com/couchbase/code_snippets/Examples.java]: Sorry, @RichardSmedley; done now. These don't seem to show up  in Atom; done now

**Review by RichardSmedley — APPROVED** [review]
LGTM

**Comment by ibsoln** [comment]
Thanks for doing these Richard.

Once I get this release done I’m hoping I’ll have some breathing space; and time available  to reciprocate.

**Comment by RichardSmedley** [comment]
> Thanks for doing these Richard. Once I get this release done I’m hoping I’ll have some breathing space; and time available to reciprocate. 

You're welcome. :+1:

**Pull request merged**